packages/components/app/styles/components/accordion.scss

```diff
@@ -0,0 +1,112 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// ACCORDION
+//
+
+@use "../mixins/button" as *;
+@use "../mixins/focus-ring" as *;
+
+:root {
+  --token-accordion-item-padding: 16px;
+  --token-accordion-item-padding-right: 12px;
+}
+
+// ACCORDION COMPONENT (wrapper)
+
+.hds-accordion {
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+
+  // override default .hds-disclosure-primitive style (Q: Should primitive be refactored?)
+  .hds-accordion-item { width: 100%; }
+}
+
+// ACCORDION ITEM COMPONENT (nested child)
+
+.hds-accordion-item {
+  background: var(--token-color-surface-primary);
+  border: 1px solid;
+  border-radius: 6px;
+
+  // isInteractive=true, Entire AccordionItem is interactive:
+  &.hds-accordion-item--is-interactive {
+    border-color: transparent;
+    box-shadow: var(--token-surface-mid-box-shadow);
+
+    &:hover,
+    &.mock-hover {
+      box-shadow: var(--token-surface-high-box-shadow);
+    }
+  }
+
+  // isInteractive=false, Only nested Chevron button is interactive:
+  &.hds-accordion-item--is--not-interactive {
+    border-color: var(--token-color-border-primary);
+  }
+}
+
+// TOGGLE BLOCK
+
+.hds-accordion-item__toggle {
+  position: relative;
+  display: flex;
+  gap: 12px;
+  align-items: center;
+  padding: var(--token-accordion-item-padding) var(--token-accordion-item-padding) var(--token-accordion-item-padding) var(--token-accordion-item-padding-right);
+}
+
+.hds-accordion-item__button {
+  @include hds-focus-ring-with-pseudo-element();
+  margin: -1px; // offset border size
+  padding: 0;
+
+  &:hover { cursor: pointer; }
+
+  // entire toggle area is interactive
+  &.hds-accordion-item__button--is-not-interactive {
+    position: static;
+    background: transparent;
+    border: 1px solid transparent;
+
+    // expand button target to cover entire AccordionItem Toggle block (depending on the `@isInteractive` argument)
+    &::after {
+      position: absolute;
+      display: block;
+      border-radius: 6px;
+      content: "";
+      inset: 0;
+    }
+  }
+
+  // only chevron button area is interactive
+  &.hds-accordion-item__button--is-interactive {
+    @include hds-button();
+    @include hds-button-color-secondary();
+
+    &:focus { border-color: var(--token-color-border-strong); }
```

Comment:
Are you sure you need this? In which case?

Reply:
I'm going to work on more refactors to the styles today because I just noticed some details of the designs for the 2 variations changed from the original explorations that I hadn't noticed so I'll go through and check things like this again as part of that.

Reply:
I double-checked this line. I'm seeing the border turn blue after I click on the button to activate it:
<img width="315" alt="image" src="https://github.com/hashicorp/design-system/assets/108769823/24bc2a78-3ddd-4696-b022-541df5f0f576">

Reply:
there may be an issue with `:focus` vs `:focus-visible` (in the implementation, I mean)?

Reply:
I've been trying different things and so far can't figure out the cause. I'll experiment more after addressing other issues.

I pushed up some other styles fixes to make the 2 variants match the latest mocks and also figured out how to get the hover state to display in the Showcase

Reply:
This is my main remaining issue I think now. Still haven't been able to figure it out but I will focus on it more next.

Reply:
The border of the button _should_ be blue when focussed, to indicate the active element on the page (just as the button component does). What is missing is the additional light-blue border that resides in `hds-button-state-focus`.

The way I see it, the `hds-accordion-item__button` class should look like this (happy to provide a patch if more helpful).

```scss
.hds-accordion-item__button {
  padding: 0;

  &:hover { cursor: pointer; }

  // entire toggle area is interactive
  &.hds-accordion-item__button--is-not-interactive {
    @include hds-focus-ring-with-pseudo-element();

    position: static;
    margin: -1px; // offset border size
    background: transparent;
    border: 1px solid transparent;

    // expand button target to cover entire AccordionItem Toggle block (depending on the `@isInteractive` argument)
    &::after {
      position: absolute;
      display: block;
      border-radius: 6px;
      content: "";
      inset: 0;
    }
  }

  // only chevron button area is interactive
  &.hds-accordion-item__button--is-interactive {
    @include hds-button();

    &:focus,
    &.mock-focus {
      @include hds-button-state-focus();
    }

    // `hds-button-color-secondary` determines the focus color and needs to be placed after `hds-button-state-focus`
    @include hds-button-color-secondary();
  }

  // animate chevron icon
  &.hds-accordion-item__button--is-open {
    .flight-icon-chevron-down {
      transform: rotate(-180deg);
    }
  }

  .flight-icon-chevron-down {
    @media (prefers-reduced-motion: no-preference) {
      transition: transform 0.3s;
    }
  }
}
```

Reply:
### Before 
![accordion-focus-before](https://github.com/hashicorp/design-system/assets/788096/965d132d-b66f-488e-b642-4e3f734ff0b7)

### After
![accordion-focus-after](https://github.com/hashicorp/design-system/assets/788096/326cdfb6-9c36-4da0-aca3-1bdecf9e01a3)

Reply:
Thank you! @alex-ju Let me see...

Reply:
Works great! I pushed up the changes. @alex-ju